src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,66 @@
+@mixin square {
+  height: $heightOfDay;
+  width: $heightOfDay;
+}
+
+@function calendarWidth($widthOfDay, $calendarPadding, $day) {
+  @return ($widthOfDay * $day) + ($calendarPadding * 2) + ($gap * 6);
+}
+
+$startDay:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+
+  width: calendarWidth($widthOfDay, $calendarPadding, 7);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendarPadding;
+
+  @each $day, $number in $startDay {
+    &--start-day-#{$day} {
+      & :first-child {
+        margin-left: #{$number * 100 + $number}px;
```

Comment:
100 = $widthOfDay, so maybe it would be better to specify here variable, because if you want to change the size of the day or calendar in the future, everything should work automatically

Reply:
that's right! thank you for all suggestions